public/javascripts/cohortController.js

```diff
@@ -14,3 +14,12 @@ var cohortController = function($scope,  $http, $state) {
     });
   };
 };
+
+var cohortStudentsController = function($scope, $http, $state) {
+
+  $scope.showStudent = function(student) {
+    $scope.$parent.$parent.currentStudent = student;
```

Comment:
Interesting. I sometimes have had to do scope.apply. I think it's because
angular doesn't automatically check for variable updates. It does in http
requests, which is usually when we end up changing variables, which is why
we haven't had to constantly do scope.apply

On Fri, Apr 29, 2016 at 2:16 PM Nora Mohamed <notifications@github.com>
wrote:

> In public/javascripts/cohortController.js
> <https://github.com/davidabrahams/Spring-Initiative/pull/161#discussion_r61622532>
> :
>
> > @@ -14,3 +14,12 @@ var cohortController = function($scope,  $http, $state) {
> >      });
> >    };
> >  };
> > +
> > +var cohortStudentsController = function($scope, $http, $state) {
> > +
> > +  $scope.showStudent = function(student) {
> > +    $scope.$parent.$parent.currentStudent = student;
>
>
> https://stackoverflow.com/questions/22557599/angularjs-parent-scope-is-not-updated-in-directive-with-isolated-scope-two-wa
>
> https://stackoverflow.com/questions/19391773/angularjs-parent-scope-not-updated-in-directive-with-isolated-scope-two-way-b
> i think these address what i'm talking about?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/davidabrahams/Spring-Initiative/pull/161/files/4564c5995d55d41f88d42c58d73ffcdc14eb369d#r61622532>
>